src/test/java/com/libertymutualgroup/herman/aws/ecs/logging/LoggingServiceTest.java

```diff
@@ -16,20 +16,6 @@
 
 public class LoggingServiceTest {
 
-    @Test(expected = IllegalArgumentException.class)
```

Comment:
Could you instead test that when the Splunk url is null or empty that it doesn't blow up?

Reply:
Good thought, updated